### DISCUSSION TITLE
Added: LDM Notes for September 9th, 2020

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#3869] [LDM Notes]
https://github.com/dotnet/csharplang/blob/master/meetings/2020/LDM-2020-09-09.md

Agenda:
- Triage issues still in C# 9.0 candidate
- Triage issues in C# 10.0 candidate

**Comment by YairHalberstadt** [comment]
> defer statement https://github.com/dotnet/csharplang/issues/1398 ... As a result, we are moving this feature to Likely Never, and if we see similar significant outcry to the rejection we can reconsider this.

I would like to register my part in the significant outcry.

**Comment by YairHalberstadt** [comment]
I have a couple of PRs open for any time features in Roslyn:

https://github.com/dotnet/roslyn/pull/44388
https://github.com/dotnet/roslyn/pull/44476

Is there any chance they could be considered for the C# 10 working set?

**Reply by 333fred** [reply]
We're not done triage, so I'll make sure we look at them next time. /cc @MadsTorgersen

**Comment by MgSam** [comment]
Primary Constructors - (Forever) Coming Next Year!™ 😉

**Comment by Joe4evr** [comment]
> Champion: `base(T)` phase two
> 
> While this was part of the initial feature set for DIMs, we don't hear any particular customer complaints for this feature.

So #2881 and #3705 are not complaints about the lack of this?

**Comment by PathogenDavid** [comment]
> While this was part of the initial feature set for DIMs, we don't hear any particular customer complaints for this feature. We'll move it to X.0 until such time as we hear customer asks for it.

Personally speaking, I never complained because I thought it was coming. It's not ruining my life or anything, but I have found myself wanting it before.
